### DISCUSSION TITLE
interesting but useless feature

### DISCUSSION THREAD

**Discussion by S-PScripts** [3kh0/ext-remover#1375] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
in v131 a new feature called safety reset was introduced (https://chromeos.dev/en/releases)
it can reset Chrome settings and Chrome shortcuts, disable extensions, and delete cookies and other temporary site data. Bookmarks, history, and saved passwords won't be affected.

You can find it here: chrome://sanitize but its useless because of ERR_INVALID_URL

**Comment by SN-Reality** [comment]
is there a way to get a valid URL yet or is that still getting discovered

**Reply by S-PScripts** [reply]
i don't think you could get around this (same thing with shimless-rma)
![Screenshot 2024-12-11 17 14 55](https://github.com/user-attachments/assets/5372d4e5-9a56-40d9-b225-e67ed21f3d46)

**Comment by akabutnicer** [comment]
yeah you have to get a pathway for it

**Comment by akabutnicer** [comment]
@S-PScripts  `chrome://sanitize/sanitize_done.html.js` here refer to this

**Reply by S-PScripts** [reply]
doesnt work for me :P

**Comment by akabutnicer** [comment]
![Screenshot 2024-12-11 12 40 53 PM](https://github.com/user-attachments/assets/b638c3f0-34c1-4121-8b06-10a925c74de7)
yeah this is practically just an unresponsive dialog, forget it

**Reply by SN-Reality** [reply]
does this unenroll your chromebook?

**Reply by S-PScripts** [reply]
> SN

even if the feature worked i don't think it unenrolls

**Reply by IGETboard** [reply]
if its invalid its just a demo or something they are working on even if they have said its released.

**Comment by IGETboard** [comment]
It will work someday just not right now, it could just be something they are experimenting with. Just becasue its a unresponsive diolog dosent mean its never gogin to work it going to happen. It could just be for devs atm.

**Reply by S-PScripts** [reply]
nah it works im pretty sure but not on enrolled chromebooks

**Reply by akabutnicer** [reply]
@IGETboard nah you need the power wash setting available in settings, which enrolled users don't have access to (as of my school, and for most others)

**Comment by akabutnicer** [comment]
@S-PScripts lowkey just remove this

**Reply by S-PScripts** [reply]
yeah i've locked it now
